odxtools/multiplexer.py

```diff
@@ -236,3 +236,25 @@ def decode_from_pdu(self, decode_state: DecodeState) -> ParameterValue:
         decode_state.origin_byte_position = orig_origin
 
         return result
+
+    @override
+    def get_static_bit_length(self) -> int | None:
+        """
+        Returns the static bit length of the multiplexer structure, if determinable.
```

Comment:
insert a blank line between the first line and the rest of the description. (Most documentation tools treat the first line as the summary...)